jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -384,67 +385,95 @@ void java_object_factoryt::gen_pointer_target_init(
   size_t depth,
   update_in_placet update_in_place)
 {
-  PRECONDITION(expr.type().id()==ID_pointer);
-  PRECONDITION(update_in_place!=update_in_placet::MAY_UPDATE_IN_PLACE);
+  PRECONDITION(expr.type().id() == ID_pointer);
```

Comment:
Different reviewers seem to have different opinions on that, I've received the exact opposite comment before. :confused: I guess from now on I'll put them in separate commits and maybe squash them before merging.

Reply:
Clang format should be run on the lines each commit updates and squashed into that commit. Clang formatting of unchanged lines should either not be done, or be done in a separate commit. I think this was commented on, because this commit is for functional changes and the only change to this line in this commit is for formatting.

Reply:
Right, I used CLion's "clang-format selected lines" feature, and probably just included those two extra lines as I think CI has complained before about lines next to some lines that I changed. Will separate this in the future.